templates/index.html

```diff
@@ -209,13 +253,13 @@ <h3 class="p-heading--2">Next-generation 5G networks</h3>
         <div class="row">
           <div class="col-2 col-medium-2 col-small-2">
             {{ image (
-              url="https://assets.ubuntu.com/v1/156b0c27-UTA%20logo.png",
-              alt="",
-              width="1012",
-              height="297",
-              hi_def=True,
-              loading="lazy"
-              ) | safe
+            url="https://assets.ubuntu.com/v1/156b0c27-UTA%20logo.png",
+            alt="",
+            width="1012",
+            height="297",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
```

Comment:
Also as mentioned before, prettier doesn't handle very well Jinja, the correct indentation should be  how it was before.

Reply:
I will ask all of us to use Prettier in the future. I just need to document it first so that it's easier for you guys to also use it. The way we are doing it now is not sustainable, since everyone writes the code how they like. We should follow same rules instead and let IDE format it automatically.